src/components/organisms/Footer/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react'
+import styles from './index.module.css'
+import Markdown from '../../atoms/Markdown'
+import { useSiteMetadata } from '../../../hooks/useSiteMetadata'
+import { ReactComponent as FooterLogo } from '../../../images/FooterLogo.svg'
+import Links from './Links'
+
+export default function Footer(): ReactElement {
+  // TODO: replace placeholder footer content
+  const { footer } = useSiteMetadata()
+  const { title, subtitle, copyright } = footer
+
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.container}>
+        <div className={styles.main}>
+          <FooterLogo />
```

Comment:
issue(blocking): Let's use a more descriptive name here. Maybe something like `hashtag.svg` and simply import as `Hashtag`